src/components/20-molecules/datepicker/index.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../00-materials/styles/mixins/check-mark-animated';
+$breakpoint: (min-width: 380px);
```

Comment:
component-local special breakpoint?? Why? Why not add to offcial breakpoints? 

At least argue for your solution here, and add a short comment in the SASS, if you cannot avoid it.